openmc/opencg_compatible.py

```diff
@@ -280,6 +343,21 @@ def get_openmc_surface(opencg_surface):
 
 
 def get_compatible_opencg_surfaces(opencg_surface):
+    """Generate OpenCG surfaces that are compatible with OpenMC equivalent to an
+    OpenCG surface that is not compatible.
```

Comment:
As above, I think a simple sentence appended to this would make this method more transparent to new users (albeit, I understand OpenCG isn't even open source yet):

"For example, this method may be used to convert a 'ZSquarePrism' OpenCG surface into a collection of equivalent XPlane and YPlane OpenMC surfaces."